src/databricks/labs/remorph/install.py

```diff
@@ -50,38 +57,79 @@ def configure(self) -> MorphConfig:
             logger.debug(f"Cannot find previous installation: {err}")
         logger.info("Please answer a couple of questions to configure Remorph")
 
+        # default params
+        catalog_name = "transpiler_test"
+        schema_name = "convertor_test"
```

Comment:
Should we move the default catalog and schema names to constants under config module? Then we can refer to them in the unit tests also if needed.